### PR TITLE
Created new FeedList Component

### PR DESCRIPTION
**Pull request by kaushiknishchay**
Component to show logged In  user's public feed.

### CODE REVIEW

File: src/components/Home.jsx [hunk 1]
```diff
@@ -22,18 +25,31 @@ class Home extends Component {
     if (this.props.token && !this.props.user) {
       this.props.getInfo();
     }
+
```
Comment (Arjun-sna) [inline src/components/Home.jsx]: try not to push commented code

File: src/components/FeedsList.jsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
+/* eslint-disable react/forbid-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function FeedList(props) {
+  function timeSince(date) {
```
Comment (Arjun-sna) [inline src/components/FeedsList.jsx]: Since this is a common function, create file name Utils.js and add these functions there so that we can reuse it.

Comment (Arjun-sna) [inline src/components/FeedsList.jsx]: check if some library provides this function like `moment.js` or `lodash`

File: src/service/httpFetch.js [hunk 3]
```diff
@@ -21,6 +21,17 @@ export function getUserDetails() {
   });
 }
 
+
+export function getFeeds(username) {
```
Comment (Arjun-sna) [inline src/service/httpFetch.js]: Remove code duplication here. Make axios configuration common.

Comment (kaushiknishchay) [reply src/service/httpFetch.js]: if i create a common config object and then assign to axios, it faces an issue. 
when i sign in , the API call to fetch the user data gives 401 error bcz the common config is not refreshed and still gets the stale value of token..

**Review by Arjun-sna — APPROVED** [review]

**Pull request merged**